plugin/builtin/input/journald.go

```diff
@@ -187,7 +187,7 @@ func (plugin *JournaldInput) parseJournalEntry(line []byte) (*entry.Entry, strin
 
 	entry := plugin.NewEntry(record)
 	entry.Timestamp = time.Unix(0, timestampInt*1000) // in microseconds
-	return entry, cursor, nil
+	return entry, cursor.(string), nil
```

Comment:
Do we need to check this cast to string? I know it might be verbose, but if there's even the slightest chance for a panic here, I'd like to avoid that.

Reply:
Can do 👍